e2e/test/MessageSendE2ETests.cs

```diff
@@ -2,12 +2,9 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using Microsoft.Azure.Devices.Client;
-using Microsoft.Azure.Devices.Client.Exceptions;
```

Comment:
👏 Thank you! This allows faster exec. In general, we do want clear separation between tests!